tests/addUserUI.spec.ts

```diff
@@ -61,18 +68,45 @@ test("Verify 'Year of Birth' field placeholder and only number input on the 'Add
 test("Check 'Gender' field content on the 'Add User' page", async ({
   page,
 }) => {
-  const genderField = page.locator('xpath=//*[@id="selectGender"]');
+  const genderField = page.locator('xpath=//select[@id="selectGender"]');
   await expect(genderField).toBeVisible;
-  // TODO: update locators which use Label
+
   // checking option 1 for gender input - Male
-  await genderField.selectOption({ label: GenderOptions[1] });
-  await expect(genderField).toHaveValue(GenderOptions.Male.toString());
+  await genderField.selectOption(GenderOptions.Male.toString());
```

Comment:
move toSring() to method

Reply:
selectOption() is a built-in function, will use toString() in a method in page class